core/modules/startup/keyboard.js

```diff
@@ -0,0 +1,77 @@
+/*\
+title: $:/core/modules/startup/keyboard.js
+type: application/javascript
+module-type: startup
+
+Keyboard shortcut handling
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+// Export name and synchronous status
+exports.name = "global-keyboard";
+exports.platforms = ["browser"];
+exports.after = ["startup"];
+exports.synchronous = true;
+
+exports.startup = function() {
+	if($tw.browser) {
+		var shortcutTiddlers = $tw.wiki.filterTiddlers("[all[shadows+tiddlers]tag[$:/tags/KeyboardShortcut]!has[draft.of]]"),
+			shortcutKeysList = [],
+			shortcutActionList = [],
+			shortcutParsedList = [];
```

Comment:
As I wrote:

```
if (!shortcutTiddlers) return;
```

IMO If no shortcuts are defined, there is no need to activate any listeners, so the function "exits early"

Reply:
uups. .. The change has to be registered. ... It's a startup module. ... my bad!

Reply:
yes, that's the point of this ... trying to cache those tiddlers, keys and actions and update them through the change listener